webrtc.html

```diff
@@ -4768,7 +4768,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             nullable</dt>
             <dd>
               <p>The <code>track</code> attribute is the track that is
-              associated with this <code><a>RTCRtpSender</a></code> object.</p>
+              associated with this <code><a>RTCRtpSender</a></code> object.
+              If <code>track.stop()</code> is called, or if 
```

Comment:
Perhaps we shouldn't limit ourselves to the track.stop() case here. I guess we want this behaviour regardless how the track was stopped. E.g. camera unplugged.